src/dynarec/rv64/dynarec_rv64_helper.h

```diff
@@ -168,9 +168,9 @@
         ed = i;                                                                               \
         wb1 = 1;                                                                              \
     }
-// GETEW will use i for ed, and can use r3 for wback.
+// GETEW will use i for ed, and *may* use x3 for wback.
 #define GETEW(i, D) GETEWW(x3, i, D)
-// GETSEW will use i for ed, and can use r3 for wback. This is the Signed version
+// GETSEW will use i for ed, and *may* use x3 for wback. This is the Signed version
```

Comment:
Add another  parameter w for wback, so it’s more obvious which register is used.

Reply:
Never mind, there are more macros like this across all 3 backends.